dockerfile_functions.sh

```diff
@@ -280,11 +337,23 @@ EOI
 # Print the JAVA_HOME and PATH.
 # Currently Java is installed at a fixed path "/opt/java/openjdk"
 print_java_env() {
-	cat >> $1 <<-EOI
+	# e.g 11 or 8
+	version=$(echo $file | cut -f1 -d"/")
+	os=$4
+	if [ "$os" == "windows" ] && [ "$version" -gt 10 ]; then
+		cat >> $1 <<-EOI
+
+# https://docs.oracle.com/javase/10/tools/jshell.htm
+# https://en.wikipedia.org/wiki/JShell
+CMD ["jshell"]
```

Comment:
Any reason why we need `jshell` ? Want to keep it consistent for all OSes

Reply:
removed